config/settings/types.py

```diff
@@ -14,3 +14,4 @@ class HawkScope(Enum):
     public_company = auto()
     data_flow_api = auto()
     metadata = auto()
+    omis = auto()
```

Comment:
Maybe `public_omis` would make it clearer that's it's not all of OMIS?
```suggestion
    public_omis = auto()
```

Reply:
Good idea! Thank you